### PR TITLE
fix: fix some issues

### PR DESCRIPTION
**Pull request by hansemannn**
https://jira.appcelerator.org/browse/MOD-2542

- [x] Fix a crash that occurred when running an app with iOS < 13
- [x] Add missing manifest architectures
- [x] Handle all cancel error codes to fire the "cancel" event instead the "error" one
- [x] Silent some unrelated warnings from the SDK (tracked on TIMOB already)
- [x] Bump to 1.1.1

### CODE REVIEW

**Comment by hansemannn** [comment]
cc @sgtcoolguy, without these fixes, no one will be able to include this on a project with min-iOS-version < 13.

**Review by vijaysingh-axway — APPROVED** [review]
Looks good.

**Comment by vijaysingh-axway** [comment]
@sgtcoolguy Build is failing for this. Please have a look. Thanks!

File: ios/manifest [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # this is your module manifest and used by Titanium
 # during compilation, packaging, distribution, etc.
 #
-version: 1.1.0
+version: 1.1.1
 apiversion: 2
 architectures: arm64 x86_64
```
Comment (ewanharris) [inline ios/manifest]:
```suggestion
 architectures: arm64 x86_64 i386 armv7
```

Build is erroring that i386, and armv7 are built into the binary, but not included in the manifest

```
[2019-09-13T06:58:44.206Z] [ERROR] : There is discrepancy between the architectures specified in module manifest and compiled binary.
[2019-09-13T06:58:44.206Z] [ERROR] : Architectures in manifest: arm64, x86_64
[2019-09-13T06:58:44.206Z] [ERROR] : Compiled binary architectures: i386, x86_64, armv7, arm64
[2019-09-13T06:58:44.206Z] [ERROR] : Please update manifest to match module binary architectures.
```

**Pull request merged**